src/general.js

```diff
@@ -42,6 +44,22 @@ if (comments_div) {
     );
 }
 
+/**
+ * Notify the user that other languages may exist from their current browser. Ie if someone visits the English site,
+ * but the browser is configured in German, lets notify them to change the lang
+ * @param  {string} preferred_language - bcp47 substring of language. e.g en-US becomes en
+ * @param  {string} website_language - bcp47 substring of current website language eg en-US becomes en
+ */
+function notifyOtherLanguages(preferred_language, website_language) {
+    if (!preferred_language || !website_language) return;
+    let recommend_language = t_r('recommend-language', preferred_language || LOCALE);
+    flash(
+        <FlashMessage type="info" duration={15000}>
```

Comment:
I think we can have the duration a lot longer (maybe even `INT_MAX`?) as this is only shown once. I was pretty annoyed in my testing that it disappeared so quickly and that will probably be a lot more annoying for a user who doesn't already know what the notification is about.

Reply:
Tried this with Number.MAX_SAFE_INTEGER  and Number.MAX_VALUE, both didn't work

Reply:
Updated it to now 90 seconds

Reply:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#maximum_delay_value), the max value for `setTimeout()` (which `FlashMessage` is using to dismiss itself) is `2147483647` which is a lot smaller than `Number.MAX_SAFE_INTEGER` (`9007199254740991`) but is actually C's `INT_MAX`, so I wasn't completely off. :D 

Please either hardcode the duration as `2^31-1` or change `FlashMessage` to interpret a duration of `-1` as "never dismiss automatically".